semantic/src/themes/default/globals/site.variables

```diff
@@ -343,14 +343,14 @@
   This rounds @size values to the closest pixel then expresses that value in (r)em.
   This ensures all size values round to exact pixels
 */
-@mini            : unit( round(@miniSize * @emSize) / @emSize, rem);
-@tiny            : unit( round(@tinySize * @emSize) / @emSize, rem);
-@small           : unit( round(@smallSize * @emSize) / @emSize, rem);
-@medium          : unit( round(@mediumSize * @emSize) / @emSize, rem);
-@large           : unit( round(@largeSize * @emSize) / @emSize, rem);
-@big             : unit( round(@bigSize * @emSize) / @emSize, rem);
-@huge            : unit( round(@hugeSize * @emSize) / @emSize, rem);
-@massive         : unit( round(@massiveSize * @emSize) / @emSize, rem);
+@mini            : unit( round(@miniSize * @emSize) / @emSize, em);
+@tiny            : unit( round(@tinySize * @emSize) / @emSize, em);
+@small           : unit( round(@smallSize * @emSize) / @emSize, em);
+@medium          : unit( round(@mediumSize * @emSize) / @emSize, em);
+@large           : unit( round(@largeSize * @emSize) / @emSize, em);
+@big             : unit( round(@bigSize * @emSize) / @emSize, em);
+@huge            : unit( round(@hugeSize * @emSize) / @emSize, em);
+@massive         : unit( round(@massiveSize * @emSize) / @emSize, em);
```

Comment:
Looks like the same definitions as in site.overrides. Do we need both definitions?